### PR TITLE
Improve landing page copy

### PR DESCRIPTION
**Pull request by mbmjertan**
# Description

After some discussion with @dingo-d, I've taken the liberty of rewriting the landing page copy a bit, as I noticed there was room to improve. 

Looking forward to feedback here!

### CODE REVIEW

File: website/src/theme/sections/features/features.js [hunk 1]
```diff
@@ -10,57 +10,37 @@ export default function Features() {
...
+      title: 'A build process to love',
+      desc: 'Eightshift Development kit comes with Webpack preconfigured. Build your JavaScript and CSS assets without breaking a sweat. Vendor prefixes? Polyfills? SCSS compilation? Asset minification? All taken care of.'
     },
     {
-      icon: 'yellow',
-      title: 'Easy SASS media queries',
-      desc: 'Build your application with easy to use media queries that are connected to blocks.',
+      icon: 'purple',
+      title: 'As responsive as it can get',
+      desc: 'Declare breakpoints once, use them anywhere: from media queries to blocks. Override any attribute value for particular breakpoints without breaking a sweat. Need to tweak that width for tablets? Done.',
```
Comment (goranalkovic-infinum) [inline website/src/theme/sections/features/features.js:43]:
```suggestion
      desc: 'Declare breakpoints once, use them everywhere — from media queries to blocks. Override any attribute value for particular breakpoints without breaking a sweat. Need to tweak that width for tablets? Done.',
```

**Review by iobrado — APPROVED** [review]
Nice!

File: website/src/theme/sections/features/features.js [hunk 2]
```diff
@@ -10,57 +10,37 @@ export default function Features() {
...
+      title: 'First-class CSS variables',
+      desc: 'Forget about conditional classnames: build out CSS variables from attribute values and scope them to a particular block with almost no code.'
     },
     {
-      icon: 'purple',
-      title: 'PHPCS',
-      desc: 'Helps you write better code with automatic code linting for PHP, SCSS, and JS.',
+      icon: 'red',
+      title: 'A build process to love',
+      desc: 'Eightshift Development kit comes with Webpack preconfigured. Build your JavaScript and CSS assets without breaking a sweat. Vendor prefixes? Polyfills? SCSS compilation? Asset minification? All taken care of.'
```
Comment (goranalkovic-infinum) [inline website/src/theme/sections/features/features.js:38]:
```suggestion
      desc: 'Eightshift Development kit comes with Webpack preconfigured, so you can start building your JavaScript and CSS assets from the start. Vendor prefixes? Polyfills? SCSS compilation? Asset minification? All taken care of!'
```
Maybe too many _without breaking a sweat_-s in one part of the page 😄

File: website/src/theme/sections/get-started/get-started.js [hunk 3]
```diff
@@ -56,8 +56,8 @@ export default function GetStarted() {
       >
         <Heading
           componentClass={component}
-          title={'Choose one of the options to get started'}
-          subtitle={'We\'ve built a bot to automate the tedious process of setting up a new project, so you can focus on your code. Just choose where you want to start.'}
+          title={'Get started in minutes, not hours'}
+          subtitle={'Boilerplate your project in a few simple steps. Get instructions for setting up an Eightshift Development kit theme or plugin, run a command (or two) and start coding. It\'s that easy.'}
```
Comment (goranalkovic-infinum) [inline website/src/theme/sections/get-started/get-started.js:60]:
```suggestion
          subtitle={'Boilerplate your project in a few simple steps. Get instructions for setting up an Eightshift Development kit theme or plugin, run a command (or two) and start coding. It\'s that easy!'}
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]
Left a comment and a suggestion. Amazing work!

**Review by dingo-d — APPROVED** [review]
Nothing to add, apply the suggestions Goc added and it's good to go 🙂 

Great work 👍🏼

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request merged**